core/src/main/java/org/frankframework/configuration/Configuration.java

```diff
@@ -185,6 +186,20 @@ public void start() {
 		state = RunState.STARTED;
 	}
 
+	/**
+	 * Opposed to close you do not need to reconfigure the configuration
```

Comment:
```suggestion
	 * Opposed to close, you do not need to reconfigure the configuration.
```

Reply:
Het is eigenlijk ook geen javadoc, ik zal een sterretje weghalen :)